README.md

```diff
@@ -54,19 +67,42 @@ Dados Clinicos
 * Fonte: HC da Unicamp
 * Conteúdo: ?
```

Comment:
Conteúdo: Nessa projeto utilizaremos dados de início de sintomas e sobrevida dos pacientes.